### PR TITLE
test: Stories with layers exposed as controls will crash if layers are instantiated with new (as opposed to json) #1895

### PR DESCRIPTION
**Pull request by nilscb**
- Prevent storybook from copying layers created with "new" because the layer will loose its prototype (superclass) in this case. It will then not be possible to call "clone" on it which is implemented in the superclass (Layer class).

- Also fixed a missing "ZIncreasingDownwards" flag.

### CODE REVIEW

**Comment by t0oF-azpn** [comment]
> Doesn't strictly deserve the "fix" prefix in the PR title, since it only affects testing or documentation.

I completely removed the Map3DLayer: even if it was not  working any more (nothing gets displayed), client code can still depend on it...

File: typescript/packages/subsurface-viewer/src/storybook/sharedSettings.tsx [hunk 1]
```diff
@@ -98,6 +98,7 @@ export const volveWellsWithLogsLayer = {
     logrunName: "BLOCKING",
     logName: "ZONELOG",
     logColor: "Stratigraphy",
+    ZIncreasingDownwards: false,
```
Comment (t0oF-azpn) [inline typescript/packages/subsurface-viewer/src/storybook/sharedSettings.tsx:101]: If this is missing here, it should also miss above (ie. for all the layers using ./volve_wells.json, right ?

**Review by t0oF-azpn — APPROVED** [review]

**Comment by t0oF-azpn** [comment]
Hi, can you find in all the stories where this pattern happens (ie. layers created using new in the story args) to fix these ?

**Comment by nilscb** [comment]
I have checked that none of them crashes,

**Review by t0oF-azpn — APPROVED** [review]

**Review by hkfb — APPROVED** [review]

**Pull request merged**